pages/linux/arch-chroot.md

```diff
@@ -0,0 +1,24 @@
+# arch-chroot
+
+> Enhanced `chroot` command for Arch Linux installation.
+> More information: <https://man.archlinux.org/man/arch-chroot.8>.
+
+- Start interactive shell in a new root directory:
+
+`arch-chroot {{path/to/new/root}}`
+
+- Specify user to run as:
+
+`arch-chroot -u {{user}} {{path/to/new/root}}`
+
+- Run a command in a new root directory:
+
+`arch-chroot {{path/to/new/root}} {{command}} {{command_arguments}}`
+
+- Generate the initial ramdisk using the preset provided by the `linux` package on new root directory:
```

Comment:
`on new root directory` doesn't really make sense here?

Reply:
@bl-ue, @owenvoke ?

Reply:
@sbrl are you sure?  This command is about changing the root directory (`arch-chroot`).

Reply:
Since this is (IIUC) just a more specific case of the previous example, I'm not sure it's worth the trouble of adding it here. I see a few options:

- Remove this example and let the generic one above provide the template for it
- Keep it, but don't attempt to explain what `mkinitcpio --preset linux` does, and why one might want to run it here (as they do [in the Arch wiki](https://wiki.archlinux.org/index.php/chroot#Run_a_single_command_and_exit))
- Keep it and edit the description to provide some context of why one would run this command. For example:
  ```suggestion
  - Generate the initial ramdisk for the new installation using the `linux` preset:
  ```

If this is indeed a common use of `arch-chroot`, it might make sense to keep it.

Reply:
Well, I think the `mkinitcpio` is not commonly used in this way. So, I can remove it.